src/util/std_types.h

```diff
@@ -1623,24 +1466,20 @@ class range_typet:public typet
   void set_to(const mp_integer &to);
 };
 
-/*! \brief Cast a generic typet to a \ref range_typet
- *
- * This is an unchecked conversion. \a type must be known to be \ref
- * range_typet.
- *
- * \param type Source type
- * \return Object of type \ref range_typet
- *
- * \ingroup gr_std_types
-*/
+/// \brief Cast a generic typet to a \ref range_typet
+///
+/// This is an unchecked conversion. \param type must be known to be \ref
+/// range_typet.
+///
+/// \param type Source type.
+/// \return Object of type \ref range_typet.
 inline const range_typet &to_range_type(const typet &type)
 {
   PRECONDITION(type.id()==ID_range);
   return static_cast<const range_typet &>(type);
 }
 
-/*! \brief A constant-size array type
-*/
+/// A constant-size array type
```

Comment:
How is this different from `array_typet`?

Reply:
They are the same in functionality it seems, `vector_typet` represents C vector type.

Reply:
Maybe worth pointing to `remove_vector`, which genuinely transforms all uses of `vector_typet` to `array_typet`.